README.md

```diff
@@ -12,7 +12,7 @@ Our solution is to make the `gruntwork-install` tool open source and to publish
 script that anyone can use to install `gruntwork-install` itself. To use it, execute the following:
 
 ```
-curl -Ls https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/master/bootstrap-gruntwork-installer.sh | bash /dev/stdin --version v0.0.20
+curl -LsS https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/master/bootstrap-gruntwork-installer.sh | bash /dev/stdin --version v0.0.20
```

Comment:
You might actually consider using `curl --silent --fail --show-error --location ...`. This will make `curl` return a non-zero exit code if it doesn't receive an `HTTP 20X` response from the server. I also prefer to use the fully written out options for readability, but that's more stylistic than a best practice.

Reply:
We use the short flags here since this is supposed to be a copy/paste one-liner that's as short as possible. But otherwise, totally agree with your approach :)

Reply:
Ah, that's a good point. But we should still probably use `--fail` or `-f` to make sure that a failed `curl` calls returns a non-zero exit code.

Reply:
Also a good practice, but I don't think it'll matter with the pipe.